docs/_user_guide/best_practices_migrating_from_jupyter.md

```diff
@@ -11,35 +18,29 @@ updateDate: 2023-12-18
 
 # Migrating Processes From JupyterHub to Yen-Slurm
 
-JupyterHub and the interactive Yens are a great resource for developing and debugging code, but is not intended to be final stop for your research computing needs.  If your process requires more resources than the technical limits of JupyterHub and Yen1-5, migrating your process to the <a href="/yen/scheduler.html">`yen-slurm` scheduler</a> will allow you to access more resources.
+JupyterHub and the interactive Yens are a great resource for developing and debugging code, but is not intended to be final stop for your research computing needs.  If your process requires more resources than the [technical limits](/_policies/user_limits/){:target="_blank"} of JupyterHub and Yen1-5, migrating your process to the [`yen-slurm`](/_user_guide/slurm/){:target="_blank"} scheduler</a> will allow you to access more resources.
 
-# How do I know if it is time to migrate my code from JupyterHub?
+## Common Reasons to Migrate Your Code From JupyterHub
 
 There are three common reasons to migrate your code from JupyterHub:
 
-1. Your code takes longer than 3 hours to run. JupyterHub has a 3 hour timeout limit on running processes. If you are baby sitting your code by keeping your laptop open and moving mouse every few hours its time to migrate.
+1. JupyterHub has a 3 hour timeout limit on running processes. If you are baby sitting your code by keeping your laptop open and moving mouse every few hours its time to migrate.
 
 2. Your code uses more than 48 cores or 192 GB of RAM on yens 2-5 or 12 cores or 320 GB of Ram on yen1. 
```

Comment:
`yen` and RAM spelling and I think instead of spelling the user limits, let's just link to http://localhost:8000/_policies/user_limits/ so we don't have to fix that later

Reply:
done